workflows/cloudify_system_workflows/dsl_import_resolver/resolver_with_catalog_support.py

```diff
@@ -155,15 +155,21 @@ def _create_zip(source, destination, include_folder=True):
 
     @staticmethod
     def _download_file(url, dest_path, target_filename=None):
-        with requests.get(url, stream=True, timeout=(5, None)) as resp:
-            resp.raise_for_status()
-            if not target_filename:
-                target_filename = os.path.basename(url)
-            file_path = os.path.join(dest_path, target_filename)
-            with open(file_path, 'wb') as f:
-                for chunk in resp.iter_content(chunk_size=8192):
-                    if chunk:
-                        f.write(chunk)
+        try:
+            with requests.get(url, stream=True, timeout=(5, None)) as resp:
+                resp.raise_for_status()
+                if not target_filename:
+                    target_filename = os.path.basename(url)
+                file_path = os.path.join(dest_path, target_filename)
+                with open(file_path, 'wb') as f:
+                    for chunk in resp.iter_content(chunk_size=8192):
+                        if chunk:
+                            f.write(chunk)
+        except Exception:
+            raise InvalidBlueprintImport(
+                'Couldn\'t download plugin from "{0}". Please check your '
+                'network connection and/or upload the plugin manually'.format(
```

Comment:
check my connection how? Is it a connection refused? Or a network unreachable? We can't know, because this hides the exception :)